docs/sources/panels/visualizations/table/table-field-options.md

```diff
@@ -44,7 +44,7 @@ If thresholds are set, then the field text is displayed in the appropriate thres
 
 ### Color background (gradient or solid)
 
-If thresholds are set, then the field background is displayed in the appropriate threshold color. 
+If thresholds are set, then the field background displays in the appropriate threshold color. 
```

Comment:
"Displays" is a transitive verb. It needs a direct object or to be in the passive.

Reply:
Then leave it as `is displayed`, where the doer is unimportant.